cmake/onnxruntime.cmake

```diff
@@ -74,9 +74,9 @@ target_link_libraries(onnxruntime PRIVATE
     ${PROVIDERS_CUDA}
     ${PROVIDERS_DNNL}
     ${PROVIDERS_NGRAPH}
+    ${PROVIDERS_OPENVINO}
```

Comment:
Why do you need to change it?

Reply:
This change is not needed. It was an unintentional effect of to the openvino v2.0 development process in three stages:
 1) develop a new provider with with a codename from scratch alongside existing v1 openvino provider when  strategy for releasing v2.0 was not decided yet 
2)After it was decided that the new provider replaces existing one, remove v1 openvino provider completely 
3) replace codename of the new provider with the name 'openvino' 

The change is removed now.